NEWS

```diff
@@ -11,6 +11,44 @@
 ### Features:
 ### Bugfixes:
 
+## 1.16.0 (January 21, 2024)
+### Features:
+#### UCP
+* Multiple performance improvements
+* Multiple improvements in error and debug messages
+#### UCT
+* Improved intra-node collectives performance for small messages
```

Comment:
collectives ? we are p2p library

Reply:
changed to: Adjusted rocm/ipc latency and zcopy threshold parameters